Electrometer/v4/_init.yaml

```diff
@@ -0,0 +1,18 @@
+# init configuration
+mode: 1
+fits_files_path: "~/develop/electrometerFitsFiles"
+range: -0.01
+integration_time: 0.01
+median_filter_active: false
+filter_active: true
+avg_filter_active: false
+tcp_port: 9999
+host: 127.0.0.1
```

Comment:
The `_init.yaml` file is meant to contain useful defaults that are applicable to all locations. Information regarding unit tests should live in another place. 
See:
https://tstn-020.lsst.io/#configuration-files-and-repositories
I heard there is a strategy for doing configs for indexed CSCs in a sensible way... maybe @wvreeven knows about it?